src/Modules/UngradedModules.jl

```diff
@@ -1112,11 +1247,41 @@ end
 @doc Markdown.doc"""
     SubQuo(A::MatElem{R}, B::MatElem{R}) where {R}
```

Comment:
This goes of course beyond this PR, but: do we *really* need to provide this low-level interface? Esp. since the implementation simply calls `SubModuleOfFreeModule` on both `A` and `B`. I'd much rather force the caller to do that, as it makes it clear what happens, and could then in some cases lead to further refactoring of the caller to use the high level types more instead of matrices.